grpc/src/compiler/swift_generator.cc

```diff
@@ -250,7 +257,7 @@ grpc::string Generate(grpc_generator::File *file,
   std::map<grpc::string, grpc::string> vars;
   vars["PATH"] = file->package();
   if (!file->package().empty()) { vars["PATH"].append("."); }
-  vars["ServiceName"] = service->name();
+  vars["ServiceName"] = WrapInNameSpace(service->namespace_parts(), service->name());
   vars["ACCESS"] = "public";
```

Comment:
You might want to separate this into `ServiceName` and `ServicePath` to solve the issue I mentioned in the swift paths.